README.md

```diff
@@ -34,3 +34,14 @@ Command line (using [sample.json](/sample.json)):
 ```sh
```

Comment:
On line 20 above can you change the link to /example.py? I can't comment on the actual line.